src/styles/main.scss

```diff
@@ -1,3 +1,5 @@
-body {
-  margin: 0;
-}
+@import "./untils/_variables";
```

Comment:
Typo in the import path: 'untils' should be 'utils'. Update the path to './utils/_variables' and './utils/_mixins'. 🕵️